### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by icebox827**
# Pull request for the To-Do List Project

- Config WebPack
- Create function for the projects
- Create function for the tasks
- Create each task in a separate card
- Add edit task button
- Add remove task button

### CODE REVIEW

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Invalid PR 😞 
Nice work setting up this project. Unfortunately am marking this PR as invalid because you didn't write a descriptive readme file. 
Readme's help other people understand what our project is all about hence making it professional.
Use this [template](https://github.com/microverseinc/readme-template) to get started.

### Martin Afani
#### Slack: whi25

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
# **Changes Required 🛑♻**

<hr>

Hello Team :)
This is such a great app. Well done! 🎉✔
Great use of webpack.
Excellent code logic separation from the DOM manipulation.
The project's features are all working just fine.
Descriptive PR summary and commit messages.
GIT Flow Guidelines achieved.
However, there are still a few more changes to do in the README file in order to get the approval. Keep up the great work.

<hr>

- [x] Please make sure to fix this image in the README because it's not showing up!
- [x] Kindly include the CLIs to run the project locally such as ```npm run build```
- **OPTIONAL** For starting a development localhost, add this line in the JSON package file
```"start": "webpack-dev-server --mode development --open"```

<hr>

That's it, please make the required changes, then ask for another CRR.
Good luck and happy coding. 👨‍💻💻

<hr>

Please feel free to contact me on Slack with the name "Mahmoud Bakr" if you had any questions. 🙏😊

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
-# to-do-list
\ No newline at end of file
+# Library
```
Comment (krys2fa) [inline README.md]: - [x] Please rename this to the appropriate name of the project as this can be misleading to any reviewer or contributor.

**Review by krys2fa — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on making all the requested changes and getting this milestone approved. 👍 
Please check the comment below to an issue which I feel was not enough to not get the project approved but it is still necessary and easy to fix.


You may now merge this branch into the development branch and go on with the next milestone.

💡  Do not forget to submit your project completion form after merging this branch.

All the best,
Chris

**Pull request merged**